### DISCUSSION TITLE
Showing the api response in the console

### DISCUSSION THREAD

**Discussion by Devorein** [JamesIves/fetch-api-data-action#349] [Q&A] [answered]
Hello, first of all, thank you for this amazing action. I was just wondering if there is any way to show the output of the API response in the console. Thank you.

**Comment by nikola184** [comment]
Is it posible to pass argument to endpoint, like a  ENDPOINT: https://example.com/{branch_name} ?

**Comment by JamesIves** [comment]
@Devorein Sorry for the late response. As of right now, there is not, but I think this is a reasonable ask. Let me see what I can do!

**Comment by JamesIves** [comment, marked as answer]
This is new possible in version 2, check the readme for the `debug` config option!
